src/bin/playbot.rs

```diff
@@ -95,7 +95,20 @@ impl Playbot {
     /// Parse a command sent to playbot (playbot's name needs to be stripped beforehand)
     ///
     /// Returns the response to send to the user (each line is a NOTICE)
-    fn parse_and_run(&mut self, code: &str) -> io::Result<String> {
+    fn parse_and_run(&mut self, mut code: &str) -> io::Result<String> {
+        let mut channel = DEFAULT_CHANNEL;
+
+        if code.starts_with("--") {
+            let mut parts = code.splitn(2, ' ');
+            match parts.next() {
+                Some("--stable") => { channel = ReleaseChannel::Stable },
+                Some("--beta") => { channel = ReleaseChannel::Beta },
+                Some("--nightly") => { channel = ReleaseChannel::Nightly },
+                _ => (),
```

Comment:
I think this means that `-- 1` will be allowed, right? Perhaps that could yield an error?